playbook/app/pb_kits/playbook/pb_tooltip/docs/styles.css

```diff
@@ -0,0 +1,13 @@
+.tooltip-text-truncation {
+  display: inline-block !important;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+.tooltip-text-truncation.short {
+  max-width: 65px;
+}
+
+.tooltip-text-truncation.long {
+  max-width: 180px;
+}
```

Comment:
This whole file feels unnecessary. Can you use the truncation built into our kits, and our max width props?

Reply:
In this case it is not possible. Body Kit adds the `ellipsis` considering the `-webkit-line-clamp: 1`. This means that I cannot check in JavaScript whether the text is being truncated or not, because its `clientWidth` will always be equal to the `scrollWidth`. I tried to implement it by adding the `maxWidth` prop to Body Kit, but even then, because of the way it does the clamp, it is not possible to check.

The custom CSS that I applied truncates the text based on the max-width of the element. When truncated, the `scrollWidth` is bigger than the `clientWidth` because the text is hidden. This way I can do the comparison and confirm that the element is truncated.

Reply:
@augustomallmann Can the body kit be updated to work? I really don't like having bits of the same code in multiple places. Maybe Body can use your code when set to 1 line?